website/content/docs/connect/transparent-proxy.mdx

```diff
@@ -9,88 +9,77 @@ description: |-
-   redirect them through the proxy.
-1. Inbound traffic is forced to go through the proxy to prevent unauthorized
-   direct access to the application.
+* Explicitly configure upstream services by specifying a local port to access them.
+* Change application to access `localhost:<chosen port>`.
+* Configure applications to only listen on the loopback interface to prevent unauthorized traffic from bypassing the mesh.
+    
+The following diagram shows how traffic flows through the mesh without transparent proxy enabled:
 
-#### Overview
+![Diagram demonstrating that without transparent proxy, applications must "opt in" to connecting to their dependencies through the mesh](/img/consul-connect/without-transparent-proxy.png)
 
-Transparent proxy allows users to reach other services in the service mesh while ensuring that inbound and outbound
-traffic for services in the mesh are directed through the sidecar proxy. Traffic is secured
-and only reaches intended destinations since the proxy can enforce security and policy like TLS and Service Intentions.
+Transparent proxy is available for Kubernetes environments. As part of the integration with Kubernetes, Consul registers Kubernetes Services, injects sidecar proxies, and enables traffic redirection.
 
-Previously, service mesh users would need to explicitly define upstreams for a service as a local listener on the sidecar
-proxy, and dial the local listener to reach the appropriate upstream. Users would also have to set intentions to allow
-specific services to talk to one another. Transparent proxying reduces this duplication, by determining upstreams
-implicitly from Service Intentions and imported services from a peer. Explicit upstreams are still supported in the [proxy service
-registration](/docs/connect/registration/service-registration) on VMs and via the
-[annotation](/docs/k8s/annotations-and-labels#consul-hashicorp-com-connect-service-upstreams) in Kubernetes.
+## Requirements
 
-To support transparent proxying, Consul's CLI now has a command
-[`consul connect redirect-traffic`](/commands/connect/redirect-traffic) to redirect traffic through an inbound and
-outbound listener on the sidecar. Consul also watches Service Intentions and imported services then configures the Envoy
-proxy with the appropriate upstream IPs. If the default ACL policy is "allow", then Service Intentions are not required.
-In Consul on Kubernetes, the traffic redirection command is automatically set up via an init container.
+Your network must meet the following environment and software requirements to use transparent proxy.
 
-## Prerequisites
+* Transparent proxy is available for Kubernetes environments.
+* Consul 1.10.0+
+* Consul Helm chart 0.32.0+. If you want to use the Consul CNI plugin to redirect traffic, Helm chart 0.47.0+ is required. Refer to [Enable the Consul CNI plugin](#enable-the-consul-cni-plugin) for additional information.
+* Consul on Kubernetes (`consul-k8s`) 0.26.0+
```

Comment:
I believe this was to describe how only the CNI Plugin is supported in 0.48.0 and above. @trujillo-adam I would update the versions across this PR to 0.48.0 to indicate the version the CNI plugin was introduced. Thanks!

Reply:
The only new information I added was the bit about which version of the helm chart you need to to use the plugin -- the rest is existing information that I reformatted for readability. I don't think we changed the `consul-k8s` requirement to use TP, did we? Should that requirement actually be `consul-k8s-control-plane`, instead?

Reply:
So I believe we can remove that last bullet because we have versioned docs and 0.26.0 was before 1.11.0 (we support n-2). Instead of Consul Helm chart 0.32.0+ you can just say Consul on Kubernetes 0.32.0+ since at that time we versioned everything with the same version (CLI, control plane , etc)